jenkins-slave/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM fedora:latest
+MAINTAINER http://fedoraproject.org/wiki/Cloud
+
+RUN dnf -y update && dnf clean all
+RUN dnf -y install java-1.8.0-openjdk openssh-server
```

Comment:
Hi @scollier, I have updated README file as well added clean all in every layer.
Thanks